package.json

```diff
@@ -46,15 +46,14 @@
   },
   "homepage": "https://permanent.org",
   "engines": {
-    "node": ">=12.0"
+    "node": ">=14.0"
   },
   "dependencies": {
-    "@sentry/node": "^7.50.0",
-    "aws-sdk": "^2.1373.0",
-    "body-parser": "^1.19.0",
-    "express": "^4.17.3",
-    "express-winston": "^4.2.0",
-    "joi": "^17.9.2",
+    "@sentry/node": "^5.27.3",
+    "aws-sdk": "^2.814.0",
+    "express": "^4.17.1",
+    "express-winston": "^4.1.0",
+    "joi": "^17.2.1",
```

Comment:
Why are some of these downgraded here? Oh, maybe because of the dependency changes that happened on main? Let's keep the versions on main.

Reply:
Now resolved.